### PR TITLE
Feature/updating node modules

### PR DESCRIPTION
**Pull request by alexjanousekGSA**
## Description

This PR upgrades the node modules that come up after running `npm outdated` with the exception of novuk-frontend since upgrading that package is a heavy lift and will be soon tagged for removal and replaced with USWDS. 

Additionally, the linting step had to be removed from the checks.yml due to odd errors about dependencies not being found. 

## TODO (optional)
- Replace novuk-frontend with USWDS
- Add back linting once we figure out why it can't find the dependencies

## Security Considerations
- Novuk-frontend is still outdated

### CODE REVIEW

**Comment by ccostino** [comment]
@alexjanousekGSA awesome seeing this working!  Can you please mark this ready for review and take it out of draft if it's good to go?

I'm also tagging @jonathanbobel for review in this because I want to make sure he and @heyitsmebev are able to check that  the chart dependencies are in good shape still with this.

Thank you all!

**Review by heyitsmebev — APPROVED** [review]
Works with the data chart!

File: package.json [hunk 1]
```diff
@@ -25,49 +25,49 @@
     "graceful-fs": "^4.2.11"
   },
   "dependencies": {
+    "@rollup/plugin-commonjs": "^26.0.1",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/stream": "^3.0.1",
     "@uswds/uswds": "^3.8.1",
     "cbor-js": "0.1.0",
+    "chart.js": "^4.4.3",
```
Comment (ccostino) [inline package.json:33]: It looks like `chart.js` is being added back in, but I don't believe we need it anymore, right @heyitsmebev and @jonathanbobel?

```suggestion
```

**Review by ccostino — COMMENTED** [review]
Thanks, @alexjanousekGSA!  Just one quick final check on the `chart.js` package.

File: package.json (hunk 1 above)
Comment (ccostino) [reply package.json:33]: Confirmed in Slack that we don't need this in place - @alexjanousekGSA, would you mind making sure this dependency isn't included?  After that we should be in good shape to merge. 🙂 Thanks!

Comment (alexjanousekGSA) [reply package.json]: Removed and pushed.

**Review by ccostino — APPROVED** [review]
Thanks, @alexjanousekGSA!

**Pull request merged**